aws/resource_aws_wafv2_regex_pattern_set_test.go

```diff
@@ -0,0 +1,368 @@
+package aws
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/wafv2"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+)
+
+func TestAccAwsWafv2RegexPatternSet_Basic(t *testing.T) {
+	var v wafv2.RegexPatternSet
+	regexPatternSetName := fmt.Sprintf("regex-pattern-set-%s", acctest.RandString(5))
```

Comment:
can you please change the pattern names to for all occurances:
`rName := acctest.RandomWithPrefix("tf-acc-test")`

Reply:
@DrFaust92 I changed it but now I'm getting `Error: Error creating WAFv2 RegexPatternSet: WAFDuplicateItemException: AWS WAF couldn’t perform the operation because some resource in your request is a duplicate of an existing one.`
I see it's based on time and not random enough 🤔

Reply:
I get that in windows as well sometimes. I usually lower parallel to 1 to overcome this

Reply:
That seems to work indeed, thanks.